internal/handler/wallet/sign.go

```diff
@@ -193,7 +193,7 @@ func loadWallet(nickname string) (*wallet.Wallet, middleware.Responder) {
 
 	w, err := wallet.Load(nickname)
 	if err != nil {
-		return nil, operations.NewRestWalletSignOperationInternalServerError().WithPayload(
+		return nil, operations.NewRestAccountSignOperationInternalServerError().WithPayload(
```

Comment:
If the wallet doesn't exist it's not a 500

Reply:
same as above

Reply:
https://github.com/massalabs/thyra-plugin-wallet/pull/121#discussion_r1182189175